core/templates/dev/head/components/OutcomeEditorDirective.js

```diff
@@ -63,6 +67,21 @@ oppia.directive('outcomeEditor', [function() {
           onExternalSave();
         });
 
+        $scope.getCurrentInteractionId = function() {
```

Comment:
@seanlip I'm stuck again. I can't seem to access the suppressDefaultAnswerGroupWarnings function defined in the StateResponses controller in StateResponses.js from anyplace else, even though I passed it in to answer_group_editor_directive.html as you suggested below. 

I looked it up and it seems like you just can't access a variable in one controller's scope from another controller's scope, hence the duplication of isSelfLoop. Should I be defining suppressDefaultAnswerGroupWarnings in responsesService instead?